docs/puzzles/day21.md

```diff
@@ -1,11 +1,312 @@
 import Solver from "../../../../website/src/components/Solver.js"
 
-# Day 21
+# Day 21: Dirac Dice
+by @sjrd
 
 ## Puzzle description
 
 https://adventofcode.com/2021/day/21
 
+## Modeling and parsing the input
+
+There are no more surprises here, so let us directly see the code:
+
+```scala
+type Cell = Int // from 0 to 9, to simplify computations
+
+case class Player(cell: Cell, score: Long)
+
+type Players = (Player, Player)
+
+def parseInput(input: String): Players =
+  val lines = input.split("\n")
+  (parsePlayer(lines(0)), parsePlayer(lines(1)))
+
+def parsePlayer(line: String): Player =
+  line match
+    case s"Player $num starting position: $cell" =>
+      Player(cell.toInt - 1, 0L)
+```
+
+The only thing worth nothing is that we use numbers 0 to 9 for the cells (the "spaces") instead of 1 to 10.
```

Comment:
It worth something :)

```suggestion
The only thing worth noting is that we use numbers 0 to 9 for the cells (the "spaces") instead of 1 to 10.
```